sevenzipjbindings/build.gradle

```diff
@@ -1,10 +1,19 @@
-repositories.mavenCentral()
+repositories {
+    repositories.mavenCentral()
+    maven {
+        // To be able to use a patched version of sevenzipjbinding-all-platforms
```

Comment:
I don't see much value in this comment as the groupId for sevenzipjbinding-all-platforms below indicates why we need this, what do you think?

Reply:
I wanted to make the intent super clear but I can remove it